docker/config/test.env

```diff
@@ -13,10 +13,8 @@ RAVEN_DSN=
 # Boto
 AWS_ACCESS_KEY=something
 AWS_SECRET_ACCESS_KEY=something
-# NOTE(willkg): We want boto to use connect_to_region, so we make this an
-# empty string
-AWS_HOST=
-# Use test buckets
```

Comment:
This was bogus. We really want to use the default boto ConnectionContext things since that's what moto will handily mock out. So this fixes that.